### ISSUE TITLE
Configuration parse error when using tilde in path

### ISSUE THREAD

**Issue by buzzert** [sinclairtarget/um#8]
If I set my `pages_directory = ~/Sync/um`, I get the following error:
```
Unable to parse configuration file line 2: 'pages_directory = ~/Sync/um', skipping
Your configuration file is /Users/james/.um/umconfig
/Users/james/.um/pages
```

um should be able to expand tilde in paths. My home directory is in a different location on my mac vs my Linux box.

**Comment by duffn** [comment]
That error comes from here because an entry in umconfig that starts with a tilde does not match this regex. https://github.com/sinclairtarget/um/blob/198ac377925e35979ccd625f677d93249e14549e/lib/um/umconfig.rb#L88

**Comment by sinclairtarget** [comment]
Fixed by fa15bbe324cd8689a5c950a7e319a4baf0e424a6. Available in the 3.1.0 release!
